src/migration/1733987625520-UserDefaultFeed.ts

```diff
@@ -0,0 +1,25 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class UserDefaultFeed1733987625520 implements MigrationInterface {
+  name = 'UserDefaultFeed1733987625520';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "user" ADD "defaultFeedId" text`);
+    await queryRunner.query(
+      `ALTER TABLE "user" ADD CONSTRAINT "UQ_ff0f49b797aca629f81cef47610" UNIQUE ("defaultFeedId")`,
```

Comment:
why unique constraint?

Reply:
Ah missed it, just something ORM generated I guess 😅